templates/EbookPlaceholderForm.php

```diff
@@ -0,0 +1,128 @@
+<?
+/**
+ * @var ?Ebook $ebook
+ */
+
+if($ebook === null){
+	$ebook = new Ebook();
+	//$ebook->EbookPlaceholder = new EbookPlaceholder();
+}
```

Comment:
Pattern is `$ebook = $ebook ?? new Ebook()`.

Reply:
Done. I might have to change this again when I get the session and error handling working like on the artworks form.